risedev.yml

```diff
@@ -250,7 +227,7 @@ risedev:
     - use: compute-node
       port: 5689
       exporter-port: 1224
-    - use: frontend-legacy
+    - use: frontend
```

Comment:
ci-3node is replaced by `ci-3cn-1fe`. This configuration won't be used in CI, and can be removed.